src/js/tabs/configuration.js

```diff
@@ -507,6 +526,31 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
                 self.analyticsChanges['BidirectionalDshot'] = newValue;
 
                 MOTOR_CONFIG.use_dshot_telemetry = value;
+
+                FILTER_CONFIG.dyn_notch_width_percent = self.previousFilterDynWidth;
+                FILTER_CONFIG.dyn_notch_q = self.previousFilterDynQ;
+
+                if (FILTER_CONFIG.gyro_rpm_notch_harmonics !== 0) { // if rpm filter is active
+                    if (value && !self.previousDshotBidir) {
+                        FILTER_CONFIG.dyn_notch_width_percent = FILTER_DEFAULT.dyn_notch_width_percent_rpm;
+                        FILTER_CONFIG.dyn_notch_q = FILTER_DEFAULT.dyn_notch_q_rpm;
+                    } else if (!value && self.previousDshotBidir) {
+                        FILTER_CONFIG.dyn_notch_width_percent = FILTER_DEFAULT.dyn_notch_width_percent;
+                        FILTER_CONFIG.dyn_notch_q = FILTER_DEFAULT.dyn_notch_q;
+                    }
+                }
+
+                if (FILTER_CONFIG.dyn_notch_width_percent !== self.previousFilterDynWidth) {
```

Comment:
Showing this (and maybe even checking) should be abstracted into one function that is used in both tabs.